libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'elewa-website-consultancy-hero-section',
+  templateUrl: './consultancy-hero-section.component.html',
+  styleUrls: ['./consultancy-hero-section.component.scss'],
+})
+export class ConsultancyHeroSectionComponent {
+
+  herotexts ={
+    title: 'Learning design',
+    description: `
+      Your organisation has the potential to turn every task into an oportunity to learn.
+      From growing crop, to nursing someone to good health, to developing the next-generation technology.
+      Let's work together to make the magic happen.
+    `
+  }
+
+  buttonData = {
+    text: 'Unlock your potential ', // Fill the require
+    bgColor: 'black',
+    color: 'white',
+    hoverBgColor: 'transparent',
+  }
+
```

Comment:
To keep the component lean and ensure readability and maintainability, the 'herotexts' and 'buttonData' variable content can be moved to a Typescript file in the data module